h2o-algos/src/main/java/hex/DataInfo.java

```diff
@@ -1083,7 +1083,6 @@ public Rows rows(Chunk [] chks) {
           row.binIds[row.nBins++] = cid;
         else if(_skipMissing) {
           row.predictors_bad = true;
-          continue;
```

Comment:
Easy, this is the last line in a loop, it does not do anything, and intelliJ says it should be removed. Actually, using continue is a little bit bad idea in general. But here it's just not needed.